bgpd/bgp_zebra.c

```diff
@@ -1276,6 +1275,8 @@ void bgp_zebra_announce(struct bgp_node *rn, struct prefix *p,
 			api_nh->label_num = 1;
 			api_nh->labels[0] = label;
 		}
+		memcpy(&api_nh->rmac, &(mpinfo->attr->rmac),
```

Comment:
doesn't it make sense to test if mpinfo->attr->rmac is null before doing the copy?